docs/api-docs/in-depth-guides/adding-customer.mdx

```diff
@@ -0,0 +1,170 @@
+    "self": "<string>"
+  },
+  "meta": {
+    "size": 2,
+    "total": 2
+  }
+}
+```
+
+After you get all the parties you would filter the parties by `company_name` to find the correct ID, either by looking through the list manually or using code to automate the process.
+
+## How to add party to tracking request if you have the party ID?
+
+To add a customer to a tracking request, you need to add the party to the tracking request as a customer relationship while being created. **Note** that a party cannot be added to a tracking request that has already been created.
+
+Endpoint: **POST** - https://api.terminal49.com/v2/tracking_requests
+
+```json Request
+{
+  "data": {
+    "type": "tracking_request",
+    "attributes": {
+      "request_type": "bill_of_lading",
+      "request_number": "MEDUFR030802",
+      "ref_numbers": [
+        "PO12345",
+        "HBL12345",
+        "CUSREF1234"
+      ],
+      "shipment_tags": [
+        "camembert"
+      ],
+      "scac": "MSCU"
+    },
+    "relationships": {
+      "customer": {
+        "data": {
+          "id": "PARTY_ID",
+          "type": "party"
+        }
+      }
+    }
+  }
+}
+```
+
+After you send a **POST** request to create a tracking request, you will receive a response with the Tracking Request ID and customer relationship. You can use this tracking request ID to track the shipment.
+
+```json Response
+{
+  "data": {
+    "id": "TRACKING_REQUEST_ID",
+    "type": "tracking_request",
+    "attributes": {
+      "request_type": "bill_of_lading",
+      "request_number": "MEDUFR030802",
+      "ref_numbers": [
+        "PO12345",
+        "HBL12345",
+        "CUSREF1234"
+      ],
+      "shipment_tags": [
+        "camembert"
+      ],
+      "scac": "MSCU"
+    },
+    "relationships": {
+      "tracked_object": {
+        "data": null
+      },
+      "customer": {
+        "data": {
+          "id": "PARTY_ID",
+          "type": "party"
+        }
+      }
+    },
+    "links": {
+      "self": "/v2/tracking_requests/TRACKING_REQUEST_ID"
+    }
+  }
+}
+```
+
+## Adding party for a customer
+
+For adding a customer to a tracking request, you need to create a party first. You can create a party through the [API](/api-docs/api-reference/parties/create-a-party).
+
+Endpoint: **POST** - https://api.terminal49.com/v2/parties
+
+```json Request
+{
+  "data": {
+    "type": "party",
+    "attributes": {
+      "company_name": "COMPANY NAME"
+    }
+  }
+}
+```
+
+After you send a **POST** request to create a party, you will receive a response with the Party ID. You can use this Party ID to add the customer to a tracking request.
+
+```json Response
+{
+  "data": {
+    "id": "PARTY_ID",
+    "type": "party",
+    "attributes": {
+      "company_name": "COMPANY NAME"
+    }
+  }
+}
+```
+
+## Editing a party
+
+You can update existing parties through the [API](/api-docs/api-reference/parties/edit-a-party).
+
+Endpoint: **PATCH** - https://api.terminal49.com/v2/parties/PARTY_ID
```

Comment:
Final thing - because it auto-links this, which goes nowhere - i think you should link to the documentation instead. Same for the get party link below.